src/scriptworker/constants.py

```diff
@@ -457,39 +451,35 @@
                             "all-release-branches": (
                                 "/releases/mozilla-beta",
                                 "/releases/mozilla-release",
-                                "/releases/mozilla-esr68",
                                 "/releases/mozilla-esr78",
                                 "/projects/maple",
                             ),
                             # Limit things like pushapk to just these branches
                             "release": ("/releases/mozilla-release",),
                             "beta": ("/releases/mozilla-beta",),
                             "beta-or-release": ("/releases/mozilla-beta", "/releases/mozilla-release"),
-                            "esr": (
-                                "/releases/mozilla-esr68",
-                                "/releases/mozilla-esr78",
-                            ),
-                            "esr68": ("/releases/mozilla-esr68",),
+                            "esr": ("/releases/mozilla-esr78",),
```

Comment:
do we also need "esr78" ?

Reply:
I think we only needed esr68 for fennec-specific things.
Good eye, though.